File: app/src/main/java/com/bunker/bunker/MyToast.java
package com.bunker.bunker;

import android.content.Context;
import android.widget.Toast;
public class MyToast {
    private static Toast mToastCurrent;

    public static void EndCurrentToast(){
        if(mToastCurrent != null){
            mToastCurrent.cancel();
        }
    }

    public static void ShowToast(String str, Context pThis){
        EndCurrentToast();
        mToastCurrent = Toast.makeText(pThis, str, Toast.LENGTH_SHORT);
        mToastCurrent.show();
    }
}

File: app/src/main/java/com/bunker/bunker/MyToast.kt
package com.bunker.bunker

import android.content.Context
import android.widget.Toast

object MyToast {
  private var mToastCurrent: Toast? = null

  fun EndCurrentToast() {
    mToastCurrent?.cancel()
  }

  fun ShowToast(str: String, pThis: Context) {
    EndCurrentToast()
    mToastCurrent = Toast.makeText(pThis, str, Toast.LENGTH_SHORT)
    mToastCurrent?.show()
  }
}
